### PR TITLE
Okta Inactive Users - Initial Release

### PR DESCRIPTION
**Pull request by dfrankel33**
### Description

Initial release of Okta Inactive Users policy

### Issues Resolved

n/a

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: fsm/okta/inactive_users/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+This policy has the following input parameters required when launching the policy.
+
+- *Okta Organization Name* - The name of your Okta Organization.  For example, if you navigate to https://contoso.okta.com to login to Okta, the value in this parameter would be `contoso`.
+- *Number of Days Since Last Login* - The number of days that a user has not logged in to Okta, which should raise an incident.
+- *Email addresses to notify* - A list of email addresses to notify
+
+### Required RightScale Roles
+
+- policy_manager
```
Comment (rshade) [inline fsm/okta/inactive_users/README.md]: We don't add policy_* ones as that is assumed.

File: fsm/okta/inactive_users/okta-inactive-users.pt [hunk 2]
```diff
@@ -0,0 +1,132 @@
+name "Inactive Users - Okta"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy will identify Okta users that have not logged in within a specified number of days and deactivate the users upon approval. See the [README](https://github.com/rightscale/policy_templates/tree/master/fsm/okta/inactive_users/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+severity "medium"
+category "SaaS Management"
```
Comment (rshade) [inline fsm/okta/inactive_users/okta-inactive-users.pt]: Should this be compliance

File: fsm/okta/inactive_users/okta-inactive-users.pt [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+parameter "param_okta_org_name" do
+  label "Okta Organization Name"
+  type "string"
```
Comment (rshade) [inline fsm/okta/inactive_users/okta-inactive-users.pt]: Please add descriptions to all the params.

File: fsm/okta/inactive_users/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+### What it does
+
+This Policy Template leverages the [Okta Users API](https://developer.okta.com/docs/reference/api/users/#list-users) to identify users that have no logged in for an extended period of time, and optionally will deactivate those users upon approval.  
+
+### Prerequesites
+
+- This Policy Template requires an Okta API Token to authenticate with the Okta API.  [This tutorial](https://developer.okta.com/docs/guides/create-an-api-token/overview/) describes how to create a new Token.
+- The Okta API Token must then be stored as a RightScale Credential in the account in which this policy will be applied. The credential must be named `OKTA_API_KEY`.
+
```
Comment (srpomeroy) [inline fsm/okta/inactive_users/README.md]: Any special permissions needed in Okta, or as long as you can create an API Token you can manage users?

**Comment by cdwilhelm** [comment]
@dfrankel33  I'm wondering where this originated from?

File: fsm/okta/inactive_users/README.md (hunk 4 above)
Comment (dfrankel33) [reply fsm/okta/inactive_users/README.md]: You have to be an admin to access that portal.  fyi it says so in that docs link

File: fsm/okta/inactive_users/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+## Inactive Users - Okta
+
+**As a best practice, this policy should only be applied to the Master Account, and not to each individual RightScale Account.**
+
+### What it does
+
+This Policy Template leverages the [Okta Users API](https://developer.okta.com/docs/reference/api/users/#list-users) to identify users that have no logged in for an extended period of time, and optionally will deactivate those users upon approval.
```
Comment (cdwilhelm) [inline fsm/okta/inactive_users/README.md]: should be have NOT logged in.

Comment (dfrankel33) [reply fsm/okta/inactive_users/README.md]: fixed

**Pull request merged**